packages/manager/src/features/OneClickApps/oneClickApps.ts

```diff
@@ -277,6 +320,29 @@ export const oneClickApps: OCA[] = [
       'All-in-one database deployment, management, and monitoring system.',
     website: 'https://docs.severalnines.com/docs/clustercontrol/',
   },
+  {
+    alt_description:
+      'Highly available, five-node enterprise NoSQL database cluster.',
+    alt_name: 'Couchbase Enterprise Server',
+    categories: ['Databases'],
+    colors: {
+      end: 'EC1018',
+      start: 'fffff',
+    },
+    description: `Couchbase Enterprise Server is a high-performance NoSQL database, built for scale. Couchbase Server is designed with memory-first architecture, built-in cache and workload isolation.`,
+    logo_url: 'couchbase.svg',
+    name: 'Couchbase Enterprise Server Cluster',
```

Comment:
I think this needs to be 

```suggestion
    name: 'Couchbase Cluster',
```



so that it matches what is in `packages/manager/src/features/StackScripts/stackScriptUtils.ts`



This should fix the failing test

Reply:
Linode Create v2 bases everything off the stackscrpt ID so Linode Create v2 is actually working fine.